### PR TITLE
Medical update - Muscle wound, Splints, Bandages, Closer Examine

### PR DESCRIPTION
**Pull request by Azarak**
## About The Pull Request
Adds splints and removes splint functionality from gauze
Adds a new muscle wound which can roll at random instead of other wounds dealt by brute damage (net more wounds from brute on average now)
You can cure the muscle wound by resting, sleeping or getting it splinted.
Adds cool overlays for having your limbs splinted or bandaged
dreamseeker_2020-12-26_03-24-31
Closer examine now allows you to see which parts of the examined person are damaged, aswell as to interact with splints and bandages
You can make improvised splints with wood and cotton, or tribal with bone and sinew
Most wounds are now a bit slower to heal

FOR REVIEWERS: There's lots of line changes because I moved 4 files for modularity

## Changelog
:cl:
/:cl:

### CODE REVIEW

File: code/__DEFINES/misc.dm [hunk 1]
```diff
@@ -41,7 +42,7 @@
 #define BODY_FRONT_LAYER		3
 #define HALO_LAYER				2		//blood cult ascended halo, because there's currently no better solution for adding/removing
 #define FIRE_LAYER				1		//If you're on fire
-#define TOTAL_LAYERS			28		//KEEP THIS UP-TO-DATE OR SHIT WILL BREAK ;_;
+#define TOTAL_LAYERS			29		//KEEP THIS UP-TO-DATE OR SHIT WILL BREAK ;_;
```
Comment (Gandalf2k15) [inline code/__DEFINES/misc.dm]: Maybe give an indication as to what has happened here

**Comment by SpookyTheFox** [comment]
It does not appear that splints have been given to medical borgs, like they were with bobmed. If I am reading this right though, they are not strictly required, since bone breaks can still be easily addressed by medical borg's ability to do surgery on any surface, and simply go through the bone gel or bone setter procedure for break conditions, and tell the patient to take a nap for muscle sprain or damage. Is that correct? Or, will medical borg need to receive a splint tool?

File: code/__DEFINES/misc.dm (hunk 1 above)
Comment (Azarak) [reply code/__DEFINES/misc.dm]: ah yes i've yet to comment that part, i wasnt sure how as I moved a ton of layers

**Comment by Azarak** [comment]
> 
> 
> It does not appear that splints have been given to medical borgs, like they were with bobmed. If I am reading this right though, they are not strictly required, since bone breaks can still be easily addressed by medical borg's ability to do surgery on any surface, and simply go through the bone gel or bone setter procedure for break conditions, and tell the patient to take a nap for muscle sprain or damage. Is that correct? Or, will medical borg need to receive a splint tool?

Splints arent essential to curing any wound as of yet. Do borgs get the gauze? Perhaps it wouldnt hurt to give them splints if they have the gauze

**Comment by SpookyTheFox** [comment]
> 
> 
> > It does not appear that splints have been given to medical borgs, like they were with bobmed. If I am reading this right though, they are not strictly required, since bone breaks can still be easily addressed by medical borg's ability to do surgery on any surface, and simply go through the bone gel or bone setter procedure for break conditions, and tell the patient to take a nap for muscle sprain or damage. Is that correct? Or, will medical borg need to receive a splint tool?
> 
> Splints arent essential to curing any wound as of yet. Do borgs get the gauze? Perhaps it wouldnt hurt to give them splints if they have the gauze

They do have gauze. Currently the only other wound difficulties I am aware of with borg are burns. It might have a similar work around that I am unaware of, but the health analyzer hint for second degree burns is to apply ointment or regenerative mesh, neither of which a borg has. Third degree burns require excision surgery followed by normal bandages, and thus seem covered right now. I think anyway, I've only done it like once or twice on the current medical system.

**Comment by hugmeorelse** [comment]
TG may be interested in this, so this may be merged upstream.

**Comment by Azarak** [comment]
Turns out not

**Review by Gandalf2k15 — APPROVED** [review]
This seems to be okay, there are no GLARING gamebreaking bugs that I can see. It's a big module, we can fix them as we go along if we discover any, it's been TM'd for a while too. It's time to merge this so we can actually do other things.

**Pull request merged**